requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 click>=7.0
 ruamel.yaml>=0.15.89
 flake8>=3.7.7
```

Comment:
Why do we pin two different flake8 versions in requirements-precommit and requirements-dev?

Reply:
This is not meant to be in this file.
I'll remove this.